### PR TITLE
(DOCSP-17936) Update Many TS Example

### PR DESCRIPTION
**Pull request by biniona-mongodb**
## Pull Request Info
Add TS Variant for Update Many Usage Example

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-17936

### Docs staging link (requires sign-in on MongoDB Corp SSO):
http://docs-mongodborg-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-17936-Update-Many/usage-examples/updateMany/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### CODE REVIEW

File: source/code-snippets/usage-examples/updateMany.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  G = "G",
+  PG = "PG",
+  PG_13 = "PG-13",
+  R = "R",
+  NR = "NOT RATED",
+}
+
+interface Movies {
+  rated: Rating;
+  random_number?: number;
```
Comment (cbush) [inline source/code-snippets/usage-examples/updateMany.ts:19]: Is "random number" overly contrived? I don't know what this is for in the context of a Movie with ratings.

Also, do you mean "Movie" (singular) or Movies?

File: source/usage-examples/updateMany.txt [hunk 2]
```diff
@@ -30,6 +30,24 @@ Example
...
+         :linenos:
+ 
+   .. tab:: TypeScript
+      :tabid: typescript
+ 
+      .. literalinclude:: /code-snippets/usage-examples/updateMany.ts
+         :language: typescript
+         :linenos:
+
+If you run the example above, you should see output that resembles the following:
```
Comment (cbush) [inline source/usage-examples/updateMany.txt:49]: I feel like I've commented before that "resembles the following:" feels stuffy and could be simplified:

>...output like:

>...output similar to:

>...output resembling:

**Review by cbush — APPROVED** [review]
Looks good. Minor comments.

File: source/code-snippets/usage-examples/updateMany.ts (hunk 1 above)
Comment (biniona-mongodb) [reply source/code-snippets/usage-examples/updateMany.ts:19]: Good catch, I agree "Movies" should be "Movie", I have added an item to this[ clean up tasks ticket](https://jira.mongodb.org/browse/DOCSP-18084) for making interface names singular when appropriate.

File: source/usage-examples/updateMany.txt (hunk 2 above)
Comment (biniona-mongodb) [reply source/usage-examples/updateMany.txt:49]: I agree resembles the following feels stuffy. I think we accept both forms in drivers docs, but as I also prefer this phrasing I'll change it to "output like".

**Comment by biniona-mongodb** [comment]
Hi @mongodb/dbx-node, could I have a review for this TypeScript Usage Example?

**Review by dariakp — APPROVED** [review]
LGTM

**Pull request merged**